app/assets/css/main.css

```diff
@@ -571,6 +572,17 @@ Context/Local Nav
 .year_divider li a {
   padding-left: 1.5em;
 }
+.context-nav.navbar-fixed {
+  position: fixed;
+  top: 7.7em;
+  left: 4.5em;
+}
+@-moz-document url-prefix() {
```

Comment:
even easier is to remove the left altogether since it shouldn't move in the horizontal direction.